src/TaskRouter.cpp

```diff
@@ -33,6 +33,11 @@ bool RouterTask::loop(System &system) {
   // do routing
   if (!_fromModem.empty()) {
     std::shared_ptr<APRSMessage> modemMsg = _fromModem.getElement();
+    std::shared_ptr<APRSMessage> DataMQTT = modemMsg;
```

Comment:
you can use directly the variable `modemMsg`

Reply:
Removed it. Was an leftover from testing.